src/app/dashboard/student-preview-card/student-preview-card.component.ts

```diff
@@ -102,10 +113,11 @@ export class StudentPreviewCardComponent implements OnInit {
     }
     return Array(4).fill('inactive');
   }
-}
 
-class AssignedProjectPreference {
-  name: string;
-  priority: number;
-  assigned: boolean;
+  showPersonDetails(student: Student) {
+    const person = this.teamService.persons.find(person => person.tumId === student.id);
```

Comment:
I am not 100% sure but the console showed an undefined error. I think it was this component. You can also fix this by using the `?` --> `this.teamService.persons?.find(...`